### PR TITLE
Add rvv testcase

### PR DESCRIPTION
**Pull request by joii2020**
Add test case:
vredand.vs
vredor.vs
vredxor.vs
vredminu.vs
vredmin.vs
vredmaxu.vs
vredmax.vs
vwredsumu.vs
vwredsum.vs

TODO. There are some problems with ckb-vm, and need to wait merge.

### CODE REVIEW

File: cases/src/misc.rs [hunk 1]
```diff
@@ -269,3 +269,71 @@ pub fn compress_into_bits(data: &[u8]) -> [u8; 256] {
...
+            true
+        } else {
+            false
+        }
+    } else {
+        let l = U256::from_little_endian(l);
+        let r = U256::from_little_endian(r);
+        l > r
+    }
+}
```
Comment (XuJiandong) [inline cases/src/misc.rs:292]: This implementation is wrong: what if they are both negative numbers?

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: cases/src/misc.rs (hunk 1 above)
Comment (joii2020) [reply cases/src/misc.rs:292]: When 16bit, 0xFFFF is -1, 0xFFFE is -2

Comment (XuJiandong) [reply cases/src/misc.rs:292]: It's OK. But I think we can use eint instead now.

Comment (joii2020) [reply cases/src/misc.rs:292]: OK

**Pull request merged**